pkg/vsphere/simulator/host_system.go

```diff
@@ -34,9 +34,24 @@ func NewHostSystem(host mo.HostSystem) *HostSystem {
 	host.Summary.Runtime = &host.Runtime
 	host.Summary.Runtime.BootTime = &now
 
-	return &HostSystem{
+	hs := &HostSystem{
 		HostSystem: host,
 	}
+
+	config := []struct {
```

Comment:
If you look at HostSystem.ConfigManager, there are 36 of these fields, we'll end up populating a few more of them.  The slice is just a table for these mappings so we don't have to duplicate the code inside the for loop x HostSystem.ConfigManager fields we want to populate.